src/page.js

```diff
@@ -61,13 +61,14 @@ export class Page {
   // Returns nothing.
   dispatch(data = {}) {
     const raw = this.recognize();
+    if (raw === void(0) || raw === null) {
+      throw new Error('<body> does not have a [data-page] attribute');
```

Comment:
@lucasmazza sure thing. although on "Chrome 51.0.2704.103 (64-bit)", when `[data-page]` attribute does not exist on `<body>`, `raw` would evaluate to `null`. So to stick with a single conditional, we can instead do maybe `if (!raw) { ... }`, would you prefer this instead? thanks.